OpenSim/Common/C3DFileAdapter.h

```diff
@@ -40,8 +40,8 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     /** Enumerated list of locations in which read in forces are expressed.
         %Measurement from force plates can be expressed by the C3DFileAdapter
         either at the OriginOfForcePlate (the default), CenterOfPressure, or
-        the PointOfWrenchApplication. It is an optional argument to 
-        C3DFileAdapter::readFile().
+        the PointOfWrenchApplication. You need to call  setLocationForForceExpression
+        before invoking C3DFileAdapter::read().
```

Comment:
When I first read this, I thought it was a requirement to call `setLocationForForceExpression`, but it's to change location if you don't want `OriginOfForcePlate`. Perhaps it could say `If you want to change how the measurement from force plates are expressed, call setLocationForForceExpression(ForceLocation)` before invoking `C3DFileAdapter::read().`.

Reply:
Clarified